source.py

```diff
@@ -8,21 +8,37 @@
 # ## Topic
 # *What problem are you (or your stakeholder) trying to address?*
 # 📝 <!-- Answer Below -->
+# The problem we are addressing is total energy consumption vs energy output
 
 # ## Project Question
 # *What specific question are you seeking to answer with this project?*
 # *This is not the same as the questions you ask to limit the scope of the project.*
 # 📝 <!-- Answer Below -->
+# On our trajectory (the world) how long can we really last with how we manage our energy. Is it really sustainable? Are there better decisions to be made regarding how to acquire green energy and reuse it.
 
 # ## What would an answer look like?
 # *What is your hypothesized answer to your question?*
-# 📝 <!-- Answer Below -->
+# 📝 There are many more sustainable ways to harvest energy. We will eliminate fossil fuels and operate on full green energy such as wind and solar.
 
 # ## Data Sources
 # *What 3 data sources have you identified for this project?*
 # *How are you going to relate these datasets?*
 # 📝 <!-- Answer Below -->
 
+# In[ ]:
+
+
+https://www.kaggle.com/datasets/pralabhpoudel/world-energy-consumption
+https://www.kaggle.com/datasets/anshtanwar/global-data-on-sustainable-energy
```

Comment:
This is a problem I faced myself. The class materials offer some good dataset sources but it may be tricky to find the one that works for your project.